src/properties.jl

```diff
@@ -36,7 +36,29 @@ end
 @inline GetProperties(names::Tuple{Vararg{Symbol}}) = GetProperties{names}()
 
 """
-    getproperties(names::Symbol...)
+    getproperties(object, names::Tuple{Vararg{Symbol}})
+
+Return a function that extracts a set of properties with the given `names` from an `object`,
```

Comment:
This one does not return a function (doc string copied)

Reply:
Good catch - thanks